doc/source/user/basics.types.rst

```diff
@@ -342,6 +342,34 @@ range of possible values.
     >>> np.power(100, 100, dtype=np.float64)
     1e+200
 
+.. note::
```

Comment:
Rather than putting this in a "note" under "overflow errors", I think it should be main text under a new section called "floating point precision".

Reply:
Thank you for the feedback!

I’ve moved the floating-point precision details into a new section titled "Floating point precision" as suggested.

Please let me know if further changes are needed!